protos/core.proto

```diff
@@ -157,6 +157,17 @@ message Assertion {
   reserved 4, 5, 6;
 }
 
+message Declaration {
+  string name = 1;
```

Comment:
It doesn't make sense to me to create a variable `name` here knowing that it will become deprecated (use of Target is preferred).

Reply:
I think you're right, and I think that's a good reason to block this PR on yours. (That's no issue for me, also no rush!)

Reply:
Will `name` finally stay?